nrovalidate/main.go

```diff
@@ -100,36 +118,46 @@ func validateCluster(args ProgArgs) error {
 		return err
 	}
 
-	data, err := nrovalidator.CollectData(context.TODO(), cli)
+	if !args.Quiet {
+		fmt.Fprintf(os.Stderr, "INFO>>>>: enabled validators: %s\n", strings.Join(args.Validations.List(), ","))
+	}
+
+	data, err := nrovalidator.Collect(context.TODO(), cli, args.Labels, args.Validations)
```

Comment:
I believe we can pass args as a whole instead of labels, and validation, this may enable better maintenance of the function in the future when there is a need to collect the data counting on other fields. what is the benefit of passing args in pieces instead of simply args ?

Reply:
good point. We should _not_ pass `args` because `args` is a type defined in `main` to hold the user-provided flags. So we would need a new structure in the `validator` pkg to collect the arguments. I'm 50/50 regarding this approach, sometimes is nice sometimes not really nice. In any case my threshold triggers around 2/3 arguments, so the next argument we add to `Collect` (if ever) would require us to evaluate this approach.